README.md

```diff
@@ -86,7 +86,20 @@ garm-cli pool create \
    --min-idle-runners 1 --max-runners 8 \
    --image Canonical:0001-com-ubuntu-server-jammy:22_04-lts-gen2:22.04.202206040 \
    --org=a2f1c7c8-b605-4560-adb7-79b95e2e462d \
-   --tags=azure,ubuntu \
+   --tags=azure,ubuntu,amd64 \
+   --provider-name azure
+```
+
+And another example for an arm64 (aarch64) based Linux pool:
+
+```bash
+garm-cli pool create \
```

Comment:
there was no need to specify `--os-arch=arm64` ? Do the tools get selected properly?